src/main/java/org/fungover/mmotodo/team/Team.java

```diff
@@ -37,13 +39,30 @@ public class Team {
     @UpdateTimestamp
     private LocalDateTime updated;
 
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "users")
-    private User users;
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @JoinColumn(name = "team_id")
+    private List<User> users;
+
+    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @JoinColumn(name = "team_id")
+    private List<Task> tasks;
```

Comment:
Why do you have fetch EAGER on users and LAZY on tasks?

Reply:
It's just a preference

Reply:
I believe it's better to leave it untouched, until we implement that functionality. We currently have no query/mutation of a Team entity in our task.graphql file. Nor does any other entity in that file refer to a Team via a property.

If we do however, decide in the future to implement a Team entity in our task.graphql, I believe it's best to leave it EAGER with the default graphql implementation. Because otherwise the n+1 problem will be introduced everytime someone refers to a user in their Team query. Take my opinion with a grain of salt, but I'm pretty certain that this will be the case. :)

Reply:
Isnt EAGER the one that introduces the n+1 problem? 

Checked my friend GPT: 
" To summarize, FetchType.EAGER loads associated entities immediately along with the owning entity, potentially causing the N+1 problem. FetchType.LAZY, on the other hand, loads associated entities on-demand, helping to avoid unnecessary data retrieval until needed. "